llvm/include/llvm/IR/IntrinsicsNVVM.td

```diff
@@ -768,6 +768,18 @@ let TargetPrefix = "nvvm" in {
       DefaultAttrsIntrinsic<[], [llvm_i32_ty],
                             [IntrConvergent, IntrNoMem, IntrHasSideEffects]>;
 
+  // Performance Monitor Events (pm events) intrinsics
+  // The imm-argument to the _idx variant must be
+  // within the range [0, 16).
+  def int_nvvm_pm_event_idx : NVVMBuiltin,
```

Comment:
I think we should remove NVVMBuiltin if we are not actually adding builtin support in this change.